projects/storefrontlib/src/cms-components/navigation/search-box/search-box.component.html

```diff
@@ -1,71 +1,64 @@
-      [resultTemplate]="rt"
-      [formControl]="searchBoxControl"
-      (keyup)="onKey($event)"
-      (selectItem)="selectSuggestion($event)"
-    />
+<label class="searchbox" [class.dirty]="!!searchBoxControl.value">
+  <input
+    [formControl]="searchBoxControl"
+    [placeholder]="'searchBox.searchHere' | cxTranslate"
+    aria-label="search"
+    (focus)="open(searchInput)"
+    (blur)="close($event)"
+    (keydown.escape)="close($event)"
+    (keydown.enter)="search($event)"
+    #searchInput
+  />
 
-    <!-- searchbox button desktop -->
-    <button
-      class="cx-button cx-button-desktop"
-      type="submit"
-      aria-label="Submit"
-      (click)="submitSearch()"
-      [disabled]="!searchBoxControl?.value"
-    >
-      <cx-icon [type]="iconTypes.SEARCH"></cx-icon>
-    </button>
+  <cx-icon
+    [type]="iconTypes.RESET"
+    aria-label="reset"
+    (mousedown)="clear()"
+    class="reset"
+  ></cx-icon>
+
+  <cx-icon
+    [type]="iconTypes.SEARCH"
+    aria-label="search"
+    class="search"
+  ></cx-icon>
+</label>
 
-    <!-- searchbox button mobile -->
-    <button
-      class="cx-button cx-button-mobile"
-      type="button"
-      aria-label="Search"
-      (click)="toggleMobileSearchInput()"
+<div *ngIf="(searchResults$ | async) as result" class="results">
+  <div
+    class="message"
+    [class.hidden]="!result.message"
+    [innerHTML]="result.message"
+  ></div>
+  <div class="suggestions" (mousedown)="disableClose()">
+    <a
+      *ngFor="let suggestion of result.suggestions"
+      (click)="setSearchboxValue(suggestion, searchInput)"
+      [innerHTML]="
+        'searchBox.suggestion' | cxTranslate: { suggestion: suggestion }
+      "
     >
-      <cx-icon [type]="iconTypes.SEARCH"></cx-icon>
-    </button>
+    </a>
+  </div>
 
-    <!-- searchbox results -->
-    <ng-template #rt let-suggestion="result">
-      <div
-        *ngIf="!suggestion.code; else productView"
-        class="cx-dropdown-content"
-      >
-        {{ suggestion }}
-      </div>
-      <ng-template #productView>
-        <div
-          [routerLink]="
-            {
-              route: 'product',
-              params: suggestion | stripHtml
-            } | cxUrl
-          "
-          class="cx-product"
-        >
-          <cx-media
-            [container]="suggestion.images?.PRIMARY"
-            format="product"
-            [alt]="suggestion.summary"
-          ></cx-media>
-          <div [innerHtml]="suggestion.name" class="cx-product-name">
-            {{ suggestion.name }}
-          </div>
-          <div class="cx-product-price">
-            {{ suggestion.price.formattedValue }}
-          </div>
-        </div>
-      </ng-template>
-    </ng-template>
+  <div class="products" (mousedown)="disableClose()">
+    <a
+      *ngFor="let product of result.products"
+      [routerLink]="
+        {
+          route: 'product',
+          params: product | stripHtml
+        } | cxUrl
+      "
+      (click)="close()"
```

Comment:
I would be good to either put `$event` here, or make parameter optional in `close` method